docs/basic/cloud-kernel.md

```diff
@@ -0,0 +1,124 @@
+# Using AllStarLink/app_rpt with Cloud Kernels
```

Comment:
I'm thinking that this entire page should be moved to docs/adv-topics

Reply:
Done